src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -313,6 +313,13 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         }
         CATCH_LOG()
 
+        try
+        {
+            auto processImageName{ wil::QueryFullProcessImageNameW<std::wstring>(_piClient.hProcess) };
+            _clientName = std::filesystem::path{ processImageName }.filename().wstring();
```

Comment:
Could use a std::move here.

Reply:
presumably `path{ move(image) }`; everything else is an rvalue